Dataset.py

```diff
@@ -265,7 +265,7 @@ def get_seq_order_for_epoch(self, epoch, num_seqs, get_seq_len=None):
       full_epoch = (epoch - 1) // partition_epoch + 1
     assert num_seqs > 0
     seq_index = list(range(num_seqs)); """ :type: list[int]. the real seq idx after sorting """
-    if self.seq_ordering == 'default':
+    if self.seq_ordering in ['default', 'in-order']:
```

Comment:
To be able to still do 'in-order' if default is something else.

Reply:
I don't have a big problem with this.
However, instead of commenting this here, maybe you should write the comment into the source code.
Also, I wonder: Would it make sense that some datasets have another default? Wouldn't that be confusing?

Reply:
This isn't needed if I make 'in-order' the default, so I will remove this.